csrankings-z.csv

```diff
@@ -341,6 +341,7 @@ Zinovi Rabinovich,Nanyang Technological University,http://www.zinovi.net,JwJRnmA
 Ziping Zhao,ShanghaiTech University,http://faculty.sist.shanghaitech.edu.cn/zhaoziping,fYYmZDsAAAAJ
 Ziqi Yang,Zhejiang University,https://person.zju.edu.cn/yangziqi,3xtC7FwAAAAJ
 Ziqiang Cao,Soochow University,http://web.suda.edu.cn/jsjkx/2021/0325/c129a22034/page.htm,NOSCHOLARPAGE
+Zirui Liu,University of Minnesota,https://zirui-ray-liu.github.io/,0i1w_egAAAAJ

```

Comment:
This is a disambiguation page (https://dblp.org/pid/196/8629.html). Please follow the instructions in the contributions guidelines.